### ISSUE TITLE
When you are in contenttyle singular slug getting error

### ISSUE THREAD

**Issue by trk** [AppoloDev/bolt-seo#8]
Example **contenttype**

```yaml
categories:
    name: Categories
    singular_name: Category
    slug: categories
    singular_slug: category
```

**When i visit:** `domain.ltd/categories` module working
**but when i visit :** `domain.ltd/category` getting error

![Ekran Resmi 2020-12-01 17 14 45](https://user-images.githubusercontent.com/455983/100751815-dd249400-33f8-11eb-9696-18a3e5c3e150.png)

**Comment by Fredxd** [comment]
You don't should be in listing case but in record case.
I will investigate on it.

**Comment by Fredxd** [comment]
Merged from #10 
Thanks @trk
